src/store/page.js

```diff
@@ -7,6 +7,7 @@ const types = {
   pending: 'pending',
   completed: 'completed',
   updateSet: 'updateSet',
+  setBlock: 'setBlock',
```

Comment:
No need for the store changes as we discussed, refresh should currently only refetch things and shouldn't worry about options changes.